modules/Calendar4You/CalendarUtils.php

```diff
@@ -441,12 +441,14 @@ function transferForAddIntoTitle($type, $row, $CD) {
 		$value = getTranslatedString($Cal_Data[1],'Calendar');
 	else
 		$value = $Cal_Data[1];
-
+	if ($CD["fieldname"] == "location")  {
+		$value = "<a href='https://www.google.nl/maps/place/".$row['location']."' target='_blank'>".$row['location']."</a>";
+	}
 	if ($type == "1")
 		return $Cal_Data[1];
 	else
-//		return "<br><b>".$Cal_Data[0]."</b>: ".$value;
-		return '<table><tr><th>'.$Cal_Data[0].':</th><td onmouseover="vtlib_listview.trigger(\'cell.onmouseover\', $(this))" onmouseout="vtlib_listview.trigger(\'cell.onmouseout\', $(this))">'.$value.'</td></tr></table>';
+		return '<br><b>'.$Cal_Data[0].'</b>: <span onmouseover="vtlib_listview.trigger(\'cell.onmouseover\', $(this))" onmouseout="vtlib_listview.trigger(\'cell.onmouseout\', $(this))">'.$value.'</span>';
```

Comment:
Let's move this discussion to the forum. On the thread you opened about it. I understand and have seen the problem, but I couldn't dedicate time to looking into it. I was thinking of making the layers "elevate" when you click on it or something like that. I also wanted to look into upgrading full calendar, we are using an old version and maybe a newer version does this better....